proj/hps_accel/gateware/test_filter_store.py

```diff
@@ -42,6 +42,9 @@ def reset_and_fill(self):
 
     def read_out(self):
         for i in range(0, len(self.filter_values), 4):
+            yield self.dut.next.eq(1)
```

Comment:
This is an interesting change. In the previous way "next" meant "I'm finished with that, give me a new one", whereas now "next" means "I'm ready to process, give me something". Am I understanding this correctly?

Reply:
Right, it goes from meaning "next" to more like "go". In this patch series it is still a strobe, but the next step is to convert it into a counter, telling each store to release N values to their streams.

I kept some notes in b/194738024 about how I arrived at this.

Reply:
Apologies for missing the note in the commit message.